index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <!-- custom style link -->
+<link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+     <!-- header start -->
+  <header class="navbar-container">
+        <h1>John Doe</h1>
```

Comment:
- [ ] The page title/logo should be a link and not a heading. Please make this change.